build

```diff
@@ -2,12 +2,12 @@
 
 set -xeuo pipefail
 
-yum install -y gcc
+yum install -y gcc make
 
 rm -rf sqlite sqlite3
 mkdir -p sqlite
 
-curl -fsSL https://sqlite.org/2020/sqlite-autoconf-$(echo "${GITHUB_REF}" | awk -F / '{ print $3 }').tar.gz \
+curl -fsSL https://sqlite.org/2024/sqlite-autoconf-3450100.tar.gz \
```

Comment:
It took me a bit to reason through this, but I _think_ that this previously expected `GITHUB_REF` to be a tag against this repository and that also then drove the version of SQLite that was downloaded.

As a result, we ended up with a tag (and then a release) which matched the SQLite version.

The URL year path means that this breaks over time..

Reply:
Yeah I spent like 30 minutes looking for github ref before I realized this. You would think it would refer to the sqlite release, but no....